src/source/rtl_text_plugin.js

```diff
@@ -48,6 +59,7 @@ export const setRTLTextPlugin = function(url: string, callback: ErrorCallback) {
             foregroundLoadComplete = true;
         }
     };
+    pluginStatus = status.loaded;
```

Comment:
I think this needs to be in the `_completionCallback`, as a replacement to `foregroundLoadComplete`. 

Also the `completionCallback` is invoked once per worker so we probably need to have counter that counts whether the state has been switched into `loaded` for all workers.

Reply:
I removed `foregroundLoadComplete` and moved the loaded status into the callback but it doesn't seem like we have to do anything else. The plugin doesn't throw an error.